sources/visual/SoundRect.cpp

```diff
@@ -558,8 +559,10 @@ bool SoundRect::getNormedPosition(double *positionsX, double *positionsY,
 
 
 // set name
-void SoundRect::setName(char *name)
+void SoundRect::setName(string name)
```

Comment:
Passage de chaîne sans copie

```suggestion
void SoundRect::setName(const string &name)
```